aiodogstatsd/contrib/aiohttp.py

```diff
@@ -3,7 +3,7 @@
 
 from aiohttp import web
 from aiohttp.web_app import _Middleware
```

Comment:
this one was also [moved](https://github.com/aio-libs/aiohttp/pull/5898) to `aiohttp.typedefs`, but wasn't released yet.
for future stability, I would recommend to use the same mechanic with conditional import.

Reply:
Duh, of course, sorry about that.  I should have thought about backwards compatibility.   I shall make your suggested changes.

Reply:
Ok, I've made the changes, amended the commit and pushed.

Reply:
looks good, but I am not sure if `mypy` will be satisfied. did you run `make lint` against your code?